track/gps_client.py

```diff
@@ -0,0 +1,247 @@
+"""GPS location determination via gpsd.
+
+Defines a single class GPS that encapsulates the procedures necessary to obtain the current location
+as a client of the gpsd daemon.
+
+The code in this module works on the assumption that gpsd is already configured and running on the
+system.
+
+It is also highly suggested that ntpd is also already configured and running on the system, and
+preferably is configured to use the gps module's pps signal via gpsd as its time source. In any
+case, the validation code in this module will (optionally) enforce that the gps time matches the
+system time to a reasonable degree of precision.
```

Comment:
When I ran `gps_test.py` I noted that the time reported always had no fractional seconds. Is this expected?